test_conformance/basic/test_enqueue_map.cpp

```diff
@@ -112,9 +114,9 @@ int test_enqueue_map_buffer(cl_device_id deviceID, cl_context context, cl_comman
 
         for( size_t q = 0; q < bufferSize; q++ )
         {
-            if( initialData[ q ] != finalData[ q ] )
+            if( referenceData[ q ] != finalData[ q ] )
             {
-                log_error( "ERROR: Sample %d did not validate! Got %d, expected %d\n", (int)q, (int)finalData[ q ], (int)initialData[ q ] );
+                log_error( "ERROR: Sample %d did not validate! Got %d, expected %d\n", (int)q, (int)finalData[ q ], (int)hostPtrData[ q ] );
```

Comment:
Nit-pick, can we print from `referenceData` instead of `hostPtrData`? I think it reads a little better based on the previous condition using `referenceData` and `finalData` (also the memory is not the same).

Reply:
Nice catch, will do!